bin/saya/scripts/1_world_setup.sh

```
@@ -0,0 +1,124 @@
+    echo "Error: Could not extract transaction hash or world address from the manifest file."
+    echo "Please check the contents of $TOML_FILE and ensure the migration was successful."
+    exit 1
+fi
+
+
+check_finality() {
+    local tx_hash="$1"
+    local url="$2"
+    
+    # Call sncast to get transaction status
+    local result=$(sncast --account dev tx-status "$tx_hash" --url "$url")
```

Shellcheck

[warning] Declare and assign separately to avoid masking return values.

(SC2155)